src/axom/doxygen_mainpage.md

```diff
@@ -14,6 +14,7 @@ Axom provides libraries that address common computer science needs.  It grew fro
 * @subpage primaltop provides an API for geometric primitives and computational geometry tests.
 * @subpage questtop provides an API to query point distance and position relative to meshes.
 * @subpage sidretop provides a data store with hierarchical structure.
+* @subpage sinatop ([S]imulation [In]sight and [A]nalysis) unified output library collects data directly within codes, outputting them to a common file output format co-designed with application developers and users.
```

Comment:
👍 -- I did not realize that SINA stood for Simulation Insight and Analysis !

If possible, please make this more obvious in the user docs.

Reply:
Simulation Includes Nifty Analysis?? 😄